tests/libs/estdlib/test_inet.erl

```diff
@@ -0,0 +1,36 @@
+%
+% This file is part of AtomVM.
+%
+% Copyright 2024 Paul Guyot <pguyot@kallisys.net>
+%
+% Licensed under the Apache License, Version 2.0 (the "License");
+% you may not use this file except in compliance with the License.
+% You may obtain a copy of the License at
+%
+%    http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS,
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+% See the License for the specific language governing permissions and
+% limitations under the License.
+%
+% SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+%
+
+-module(test_inet).
+
+-export([test/0]).
+
+test() ->
+    ok = test_getaddr(),
+    ok.
+
+test_getaddr() ->
+    {ok, {127, 0, 0, 1}} = inet:getaddr(localhost, inet),
+    {ok, {127, 0, 0, 1}} = inet:getaddr("localhost", inet),
+    {ok, {_, _, _, _}} = inet:getaddr("www.atomvm.net", inet),
+    {ok, {173, 48, 222, 98}} = inet:getaddr("pool-173-48-222-98.bstnma.fios.verizon.net", inet),
```

Comment:
as far as I know `example.com` is guaranteed to be static and assigned to `93.184.216.34`.

Reply:
I’m not sure about this statement and it seems example.com resolves differently here and on CI.